api/firmware/btc.go

```diff
@@ -367,3 +368,38 @@ func (device *Device) BTCRegisterScriptConfig(
 	}
 	return nil
 }
+
+// BTCSignMessage signs a Bitcoin message. The 64 byte raw signature, the recoverable ID and the 65
+// byte signature in Electrum format are returned.
+func (device *Device) BTCSignMessage(
+	coin messages.BTCCoin,
+	scriptConfig *messages.BTCScriptConfigWithKeypath,
+	message []byte,
+) ([]byte, byte, []byte, error) {
```

Comment:
You'd typically name the returned values in this case to make it more readable:

```go
(raw []byte, recid byte, electrum []byte, err error)
```

Reply:
But they are so awkward. Do you then use `return ...` or set the vars directly?

Reply:
No naked return, the method body stays as is. It's only for the docs.

https://tour.golang.org/basics/7

>  These names should be used to document the meaning of the return values.

Reply:
So, you'd never use `raw`, `recid` or `electrum` in the function body.

Reply:
:+1: